packages/common/src/adapters/track.ts

```diff
@@ -219,3 +222,119 @@ export const stemTrackMetadataFromSDK = (
     is_owned_by_user: false
   }
 }
+
+export const trackMetadataForUploadToSdk = (
+  input: TrackMetadataForUpload
+): TrackMetadata => ({
+  ...camelcaseKeys(
+    pick(input, [
```

Comment:
Heh, I don't wonder if `omit` would be less code. But I get that we're trying to specifically map the properties that are accepted by the upload schema.
The one downside I guess is that if we add properties, we have to remember to update this.